### PR TITLE
IGNITE-16176 Adds configurable Kafka requests timeout.

### PR DESCRIPTION
**Pull request by ololo3000**

### CODE REVIEW

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java [hunk 1]
```diff
@@ -84,22 +98,28 @@
     private KafkaProducer<Integer, byte[]> producer;
 
     /** Handle only primary entry flag. */
-    private final boolean onlyPrimary;
+    private boolean onlyPrimary = DFLT_IS_ONLY_PRIMARY;
 
     /** Topic name. */
-    private final String topic;
+    private String topic = DFLT_TOPIC;
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java:104]: Let's make default topic `null.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java [hunk 2]
```diff
@@ -84,22 +98,28 @@
...
     /** Kafka properties. */
-    private final Properties kafkaProps;
+    private Properties kafkaProps;
 
     /** Cache IDs. */
-    private final Set<Integer> cachesIds;
+    private Set<Integer> cachesIds;
+
+    /** Cache names. */
+    private Collection<String> cacheNames = DFLT_CACHES;
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java:116]: Let's make default caches null.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java [hunk 3]
```diff
@@ -110,37 +130,17 @@
...
-        int maxBatchSize,
-        boolean onlyPrimary,
-        Properties kafkaProps
-    ) {
-        assert caches != null && !caches.isEmpty();
-
-        this.topic = topic;
-        this.kafkaParts = kafkaParts;
-        this.onlyPrimary = onlyPrimary;
+    public IgniteToKafkaCdcStreamer(Properties kafkaProps) {
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java:141]: Can we get rid of this constructor?

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java [hunk 4]
```diff
@@ -84,22 +98,28 @@
...
+
+    /** Cache names. */
+    private Collection<String> cacheNames = DFLT_CACHES;
 
     /** Max batch size. */
-    private final int maxBatchSize;
+    private int maxBatchSize = DFLT_MAX_BATCH_SIZE;
+
+    /** The maximum time to complete Kafka related requests, in seconds. */
+    private int kafkaReqTimeout = DFLT_REQ_TIMEOUT;
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java:122]: Let's measure this timeout in milliseconds

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java [hunk 5]
```diff
@@ -194,7 +194,7 @@ public IgniteToKafkaCdcStreamer(
         if (!futs.isEmpty()) {
             try {
                 for (Future<RecordMetadata> fut : futs)
-                    fut.get(DFLT_REQ_TIMEOUT, TimeUnit.SECONDS);
+                    fut.get(kafkaReqTimeout, TimeUnit.SECONDS);
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java:197]: Let's measure this timeout in milliseconds

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java [hunk 6]
```diff
@@ -213,6 +213,16 @@ public IgniteToKafkaCdcStreamer(
 
     /** {@inheritDoc} */
     @Override public void start(MetricRegistry mreg) {
+        if (kafkaProps == null)
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java:216]: Let's use `A.notNull` here.

Please, take a look at `org.apache.ignite.internal.util.GridArgumentCheck`

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java [hunk 7]
```diff
@@ -213,6 +213,16 @@ public IgniteToKafkaCdcStreamer(
...
+            throw new IllegalStateException("The Kafka properties must be set in order to initiate a connection.");
+
+        kafkaProps.setProperty(KEY_SERIALIZER_CLASS_CONFIG, IntegerSerializer.class.getName());
+        kafkaProps.setProperty(VALUE_SERIALIZER_CLASS_CONFIG, ByteArraySerializer.class.getName());
+
+        cachesIds = cacheNames.stream()
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java:222]: We should check `A.notEmpty(cacheNames)` on start.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java [hunk 8]
```diff
@@ -114,6 +111,9 @@
     /** */
     private final AtomicLong rcvdEvts = new AtomicLong();
 
+    /** The maximum time to complete Kafka related requests, in seconds. */
+    private final int kafkaReqTimeout;
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java:115]: Let's measure timeout in milliseconds.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java [hunk 9]
```diff
@@ -211,7 +213,7 @@ private void poll(KafkaConsumer<Integer, byte[]> cnsmr) throws IgniteCheckedExce
 
         apply(F.iterator(recs, this::deserialize, true, rec -> F.isEmpty(caches) || caches.contains(rec.key())));
 
-        cnsmr.commitSync(Duration.ofSeconds(DFLT_REQ_TIMEOUT));
+        cnsmr.commitSync(Duration.ofSeconds(kafkaReqTimeout));
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java:216]: Let's measure timeout in milliseconds.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java [hunk 10]
```diff
@@ -201,7 +203,7 @@ public KafkaToIgniteCdcStreamerApplier(
      * @param cnsmr Data consumer.
      */
     private void poll(KafkaConsumer<Integer, byte[]> cnsmr) throws IgniteCheckedException {
-        ConsumerRecords<Integer, byte[]> recs = cnsmr.poll(Duration.ofSeconds(DFLT_REQ_TIMEOUT));
+        ConsumerRecords<Integer, byte[]> recs = cnsmr.poll(Duration.ofSeconds(kafkaReqTimeout));
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java:206]: Let's measure timeout in milliseconds.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java [hunk 11]
```diff
@@ -182,7 +184,7 @@ public KafkaToIgniteCdcStreamerApplier(
         finally {
             for (KafkaConsumer<Integer, byte[]> consumer : cnsmrs) {
                 try {
-                    consumer.close(Duration.ofSeconds(DFLT_REQ_TIMEOUT));
+                    consumer.close(Duration.ofSeconds(kafkaReqTimeout));
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java:187]: Let's measure timeout in milliseconds.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java (hunk 6 above)
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java:216]: Let's add A.notNull(topic) here.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java [hunk 12]
```diff
@@ -232,4 +242,139 @@ public IgniteToKafkaCdcStreamer(
...
+    }
+
+    /**
+     * Sets whether entries only from primary nodes should be handled.
+     *
+     * @param onlyPrimary Kafka whether entries only from primary nodes should be handled.
+     * @return {@code this} for chaining.
+     */
+    public IgniteToKafkaCdcStreamer setOnlyPrimary(boolean onlyPrimary) {
+        assert producer == null;
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java:258]: Let's extract this and same checks into separate method like `ensureNotStart()`.

Let's have flag `private volatile boolean started`, also.
And set it to the `true` or `false` on start and stop.

Comment (nizhikov) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java:258]: Second thought - do we have same check for `IgniteConfiguration`?
Do we really need one?

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java (hunk 1 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java]: Done. I have done the symmetric changes for KafkaToIgniteCdcStreamer too.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java (hunk 2 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java]: Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java (hunk 3 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java]: Yes. Removed.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java (hunk 4 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java:118]: Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java (hunk 5 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java]: Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java (hunk 6 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java]: Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java (hunk 7 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java]: It seems that we can check it right in the setter. Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java (hunk 8 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java:115]: Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java (hunk 9 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java]: Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java (hunk 10 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java]: Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java (hunk 11 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java]: Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java (hunk 6 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java]: Done.

I have done the symmetric check for KafkaToIgniteStreamer too.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java (hunk 12 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java]: Let's just remove this checks then. 
Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java (hunk 11 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java]: Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerConfiguration.java [hunk 13]
```diff
@@ -121,4 +127,16 @@ public void setMaxBatchSize(int maxBatchSize) {
     public void setCaches(Collection<String> caches) {
         this.caches = caches;
     }
+
+    /** */
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerConfiguration.java:131]: Please, add javadoc.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerConfiguration.java [hunk 14]
```diff
@@ -121,4 +127,16 @@ public void setMaxBatchSize(int maxBatchSize) {
...
+    public int getKafkaRequestTimeout() {
+        return kafkaReqTimeout;
+    }
+
+    /** */
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerConfiguration.java:136]: Please, add javadoc.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java [hunk 15]
```diff
@@ -213,6 +196,18 @@ public IgniteToKafkaCdcStreamer(
 
     /** {@inheritDoc} */
     @Override public void start(MetricRegistry mreg) {
+        A.notNull(kafkaProps, "Kafka properties");
+        A.notNull(topic, "Kafka topic");
+
+        kafkaProps.setProperty(KEY_SERIALIZER_CLASS_CONFIG, IntegerSerializer.class.getName());
+        kafkaProps.setProperty(VALUE_SERIALIZER_CLASS_CONFIG, ByteArraySerializer.class.getName());
+
+        cachesIds = cacheNames == null
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java:205]: We should add check `A.notEmpty(cacheNames)` to ensure caches are set.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerConfiguration.java (hunk 13 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerConfiguration.java]: Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerConfiguration.java (hunk 14 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerConfiguration.java]: Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java (hunk 15 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java]: Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java [hunk 16]
```diff
@@ -84,22 +91,28 @@
...
-    private final boolean onlyPrimary;
+    private boolean onlyPrimary = DFLT_IS_ONLY_PRIMARY;
 
     /** Topic name. */
-    private final String topic;
+    private String topic;
 
     /** Kafka topic partitions count. */
-    private final int kafkaParts;
+    private int kafkaParts = DFLT_PARTS;
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java:100]: Let's get rid of this default.
Looks like partitions count value should be provided explicitly always.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java [hunk 17]
```diff
@@ -232,4 +225,123 @@ public IgniteToKafkaCdcStreamer(
...
+     * @return {@code this} for chaining.
+     */
+    public IgniteToKafkaCdcStreamer setKafkaPartitions(int kafkaParts) {
+        this.kafkaParts = kafkaParts;
+
+        return this;
+    }
+
+    /** @return Cache names that participate in CDC. */
+    public Collection<String> getCaches() {
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java:281]: Can we get rid of all getters?

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java [hunk 18]
```diff
@@ -114,6 +111,9 @@
     /** */
     private final AtomicLong rcvdEvts = new AtomicLong();
 
+    /** The maximum time to complete Kafka related requests, in milliseconds */
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java:114]: Typo: Dot in the end required

Comment (nizhikov) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java:114]: Please, check all other javadocs, also.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java [hunk 19]
```diff
@@ -134,7 +134,8 @@ public KafkaToIgniteCdcStreamerApplier(
         int kafkaPartTo,
         Set<Integer> caches,
         int maxBatchSize,
-        AtomicBoolean stopped
+        AtomicBoolean stopped,
+        int kafkaReqTimeout
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java:138]: Looks like javadoc for the new parameter not added.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerConfiguration.java [hunk 20]
```diff
@@ -20,35 +20,41 @@
...
+    private String topic;
 
     /** Kafka partitions lower bound (inclusive). */
     private int kafkaPartsFrom = 0;
 
     /** Kafka partitions higher bound (exclusive). */
     private int kafkaPartsTo = DFLT_PARTS;
 
+    /** The maximum time to complete Kafka related requests, in milliseconds. */
+    private int kafkaReqTimeout = DFLT_KAFKA_REQ_TIMEOUT;
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerConfiguration.java:56]: Why `int`? Looks like `Duration.ofMillis` take `long` as a parameter.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java [hunk 21]
```diff
@@ -84,22 +91,28 @@
...
+
+    /** Cache names. */
+    private Collection<String> cacheNames;
 
     /** Max batch size. */
-    private final int maxBatchSize;
+    private int maxBatchSize = DFLT_MAX_BATCH_SIZE;
+
+    /** The maximum time to complete Kafka related requests, in milliseconds. */
+    private int kafkaReqTimeout = DFLT_KAFKA_REQ_TIMEOUT;
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java:115]: Why `int`? Looks like `Duration.ofMillis` take `long` as a parameter.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java (hunk 16 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java]: Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java (hunk 17 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java]: Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java (hunk 18 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java]: Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java (hunk 19 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerApplier.java]: Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerConfiguration.java (hunk 20 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/KafkaToIgniteCdcStreamerConfiguration.java]: Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java (hunk 21 above)
Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java]: Done.

Comment (ololo3000) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java]: Done.

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java [hunk 22]
```diff
@@ -47,6 +50,9 @@
 import static org.apache.ignite.cdc.IgniteToIgniteCdcStreamer.EVTS_CNT_DESC;
 import static org.apache.ignite.cdc.IgniteToIgniteCdcStreamer.LAST_EVT_TIME;
 import static org.apache.ignite.cdc.IgniteToIgniteCdcStreamer.LAST_EVT_TIME_DESC;
+import static org.apache.ignite.cdc.kafka.KafkaToIgniteCdcStreamerConfiguration.DFLT_KAFKA_REQ_TIMEOUT;
+import static org.apache.ignite.cdc.kafka.KafkaToIgniteCdcStreamerConfiguration.DFLT_MAX_BATCH_SIZE;
+import static org.apache.ignite.cdc.kafka.KafkaToIgniteCdcStreamerConfiguration.DFLT_PARTS;
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java:55]: Unused import, please, remove.

**Review by nizhikov — APPROVED** [review]

**Pull request merged**